index.css

```diff
@@ -0,0 +1,148 @@
+.container {
+  max-width: 80%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.center {
+  text-align: center;
+}
+
+.p-1 {
+  padding: 1rem;
+}
+
+.mt-1 {
+  margin-top: 0.5rem;
+}
+
+.ml-1 {
+  margin-left: 1rem;
+}
+
+.bar {
+  color: #000;
+  width: 3rem;
+  height: 5rem;
+}
+
+.button {
+  width: 7rem;
+  height: 2.5rem;
+  box-shadow: 0 2px 2px #000;
+}
+
+.add-btn {
+  margin-left: 25% !important;
+  margin-right: 0%;
+}
+
+#books {
+  margin: 6rem 0 0 0;
+}
+
+#add-books {
+  margin: 6rem 0 20rem 0;
+}
+
+.books {
+  border: 2px solid #000;
+
+  /* padding: 1rem; */
+  border-radius: 2px;
+}
+
+.input {
+  width: 30rem;
+  height: 2rem;
+  border: 2px solid #000;
+  border-radius: 6px;
+  padding: 0.5rem;
+}
+
+.book-title {
+  /* text-transform: uppercase; */
```

Comment:
- [ ] Please kindly remove commented code from your code base, as it makes your code dirty and less readable.